models/DataObject/ClassDefinition/Data/Consent.php

```diff
@@ -197,6 +197,9 @@ public function getDataFromEditmode($data, $object = null, $params = [])
             }
             $noteId = $note->getId();
         }
+        elseif ($oldData instanceof DataObject\Data\Consent) {
```

Comment:
@mcop1 could you please explain this change because it is not about CSS at all

Reply:
That´s because the noteId wasn´t always set properly, when the corresponding checkbox got (un)checked. Therefore it could happen, that when you save your changes, the noteId would still be missing, resulting in the consent note not being displayed properly (even with the fixed css).